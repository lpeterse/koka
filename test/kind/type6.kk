type int
alias id<a :: V -> V >  = a<int>
type list<a> { con Single(x:a) }
alias ok = id<list>
