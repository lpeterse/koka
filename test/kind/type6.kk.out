kind/type6/id: (V -> V) -> V
kind/type6/int: V
kind/type6/list: V -> V
kind/type6/ok: V
public alias id<a> :: (V -> V) -> V = a<int> = 1
public alias ok = id<list> = 2